src/styles/main.css

```diff
@@ -0,0 +1,4 @@
+@import "./utils/reset.css";
+@import "./utils/normalize.css";
```

Comment:
You don't need to use both `reset.css` and `normalize.css`. Choose one of them to avoid conflicts and unnecessary resets. If you want to reset all the default styles, use `reset.css`. If you want to normalize and maintain some default browser styling, use `normalize.css`. 😊